step00-start/.gitignore

```diff
@@ -1,10 +1 @@
-*.iml
```

Comment:
Why remove all of these files from the .gitignore? These look like good files to exclude.

Reply:
I accidentally copied over them when moving the modules, will restore